apache/incubator-kie-tools
# KOGITO-8824: [Dashbuilder] Support "accumulate" flag in data sets

This is a new feature for Dashbuilder datasets where it is possible to configure a dataset to keep the old values in memory.

To test: 

* Build until dashbuilder-editor and run dashbuilder-editor as a webapplication.
* Use the following YAML - make sure that the file data exists on the dataset url

```
datasets:
  - uuid: acc
    url: http://127.0.0.1:3001/data
    accumulate: true
    cacheMaxRows: 5    
pages:
  - components:  
      - settings:                    
          component: table
          refresh:
              interval: 1
          lookup:
            uuid: acc
```
* Create some dataset, here's a sample:

```
[
    ["A", 3000]
]
```
* Notice that dashbuilder constantly pulls data from the dataset, so if you modify the file you will see that the old values are kept

![image](https://user-images.githubusercontent.com/359820/223837849-d2f717f7-bc99-4f86-a5b4-a87ad025b57e.png)


You can see also in this video

https://drive.google.com/file/d/1ZwDq0jgnMeYvdS5GFuGRTrJO7T2VXxc8/view?usp=share_link
